src/styles/theme.js

```diff
@@ -1,4 +1,4 @@
-export const theme = {
+module.exports = {
```

Comment:
Gatsby doesn't support esm?

Reply:
Nope, they have some issues on GitHub related to this but they are not even in progress I think.